spinnman/messages/scp/impl/scp_utils.py

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
shouldnt this be defined as a function and placed in utilities? it seems strange to have a dict table of read address, read length = read type when its built from a bit of math that cvan be represetened as a function call